diff --git a/gframe/game.h b/gframe/game.h
index eab91f8a..22a2322a 100644
--- a/gframe/game.h
+++ b/gframe/game.h
@@ -541,6 +541,10 @@ public:
 	void WaitFrameSignal(int frame, std::unique_lock<epro::mutex>& _lck);
 	void DrawThumb(const CardDataC* cp, irr::core::vector2di pos, LFList* lflist, bool drag = false, const irr::core::recti* cliprect = nullptr, bool loadimage = true);
 	void DrawDeckBd();
+#ifdef WITH_REPLAY2VIDEO
+	void DrawDeckBdR2V();
+	static float GetDeckBdR2VScale();
+#endif
 	void SaveConfig();
 	struct RepoGui {
 		std::string path;
