diff --git a/gframe/replay_mode.h b/gframe/replay_mode.h
index f6559501..287a2ddd 100644
--- a/gframe/replay_mode.h
+++ b/gframe/replay_mode.h
@@ -1,6 +1,7 @@
 #ifndef REPLAY_MODE_H
 #define REPLAY_MODE_H
 
+#include <vector>
 #include "epro_thread.h"
 #include "dllinterface.h"
 #include "config.h"
@@ -41,6 +42,13 @@ public:
 	static bool ReadReplayResponse();
 	static int ReplayThread();
 	static int OldReplayThread();
+
+#ifdef WITH_REPLAY2VIDEO
+	static void ReplayThreadBatchWrapper();
+	static bool InitBatch();
+	static bool StepBatch();
+	static std::vector<CoreUtils::Packet>::const_iterator batch_it;
+#endif
 	static bool StartDuel();
 	static void EndDuel();
 	static void Restart(bool refresh);
